### PR TITLE
Fix quote strategy for reserved keyword in Postgres

### PR DESCRIPTION
**Pull request by j0k3r**
Check #1498 for more detail & information.

Todo:

- [x] add tests on the subscriber
- [x] add tests on the install command

### CODE REVIEW

File: src/Wallabag/CoreBundle/Tests/Command/InstallCommandTest.php [hunk 1]
```diff
@@ -185,14 +190,22 @@ public function testRunInstallCommandChooseResetSchema()
...
+             * ERROR:  source database "template1" is being accessed by other users
+             * DETAIL:  There is 1 other session using the database.
+             * STATEMENT:  CREATE DATABASE "wallabag"
+             * FATAL:  database "wallabag" does not exist
+             *
+             * http://stackoverflow.com/a/14374832/569101
+             */
+            $this->markTestSkipped('PostgreSQL spotted: can find a good way to drop current database, skipping.');
+        }
+
```
Comment (nicosomb) [inline src/Wallabag/CoreBundle/Tests/Command/InstallCommandTest.php]: maybe you can refactor to avoid duplicate lines?

**Comment by nicosomb** [comment]
:+1:

File: src/Wallabag/CoreBundle/Tests/Command/InstallCommandTest.php (hunk 1 above)
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Tests/Command/InstallCommandTest.php]: Done

**Comment by j0k3r** [comment]
Ok now sqlite test are failing. I'll check that later :disappointed:

**Comment by j0k3r** [comment]
:rocket:

**Pull request merged**